ds-collector/ds-collector

```diff
@@ -479,11 +482,13 @@ collect_via_rust() {
   export skipSudo="${skipSudo:-false}"
   export skip_dse="${skip_dse:-false}"
   export prometheus_jar="${baseDir}/${prometheus}"
+  export jmxHost
   export jmxPort
   export jmxUsername
   export jmxPassword
   export jmxSSL
-  export jmxHost
+  export jmxPlain="$(if [ "${jmxSSL}" = "true" ]; then echo "false"; else echo "true"; fi)"
```

Comment:
(trivial) Is this not going to act weird, because the `"` are closing where you don't think they are closing? Not sure it matters hugely, particularly as this is one-word: the outside `"` are unnecessary...